specs/Economics/README.md

```diff
@@ -17,25 +17,28 @@
 | - | - |
 | `INITIAL_SUPPLY` | `10**33` yoctoNEAR |
 | `NEAR` | `10**24` yoctoNEAR |
+| `INITIAL_GAS_PRICE` | `10**5` yoctoNEAR |
```

Comment:
how is this determined

Reply:
By solving gas_limit * gas_price = 0.1 N